tests/nocover/test_statistical_distribution.py

```diff
@@ -416,3 +416,117 @@ def distorted(x):
 test_integers_are_often_small = define_test(
     integers(), 0.2, lambda x: abs(x) <= 100
 )
+
+
+# This series of tests checks that the one_of() strategy flattens branches
+# correctly.  We assert that the probability of any branch is equal
+# (1/8 = 0.125), regardless of how heavily nested it is in the `one_of()`.
```

Comment:
Technically we only assert it's >= 0.1 (which is fine, but means the comment is slightly wrong).